custom_dc/load_data.sh

```diff
@@ -0,0 +1,313 @@
+#!/bin/bash
+# Script to load data into custom Data Commons using simple importer.
+# Defaults
+MODE="customdc"
+OUTPUT_DIR=".data/output_$MODE"
+USAGE="Script to load data into custom Data Commons using simple importer.
+Usage: $(basename $0) [Options]
+Options:
+  -c <file>       JSON config file for stats importer
+  -e <file>       Load environment variables from file.
```

Comment:
Retain only an optional `-e <file>` that defalut to env.list and remove all other command line options.

Reply:
made config file a direct parameter without the '-e' option.